Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -248,6 +248,13 @@ def populate(self, issue, member=None):
                 issue._version = response.get('version', '')
                 issue._keywords = response.get('keywords', [])
 
+                related_dict = dict()
```

Comment:
Should probably construct this directly instead of line-by-line:
```
issue._related = dict(
    blocks=response.get('blocks') or [],
    ...
)
```

Another question I have here: what is the type of `issue._related` supposed to be? I would expect it to be issue objects, but that's clearly not what `response.get` is supposed to be. Note that because issues are lazily populated, assuming `blocks` is a list of Bugzilla IDs, you can safely construct `Issue` objects without worrying about network requests until code starts using the issue.

Reply:
It's a dictionary where keys are the relationship and values are a list of integer bug ids. 
`issue._related = {'depends_on': [], 'blocks': [], 'regressions': [], 'regressed_by': []}`
Do you mean that we can add `self.issue(bug_id)` to the lists instead of just `bug_id`? Now that I'm thinking about it, this might be a better place to construct `Issue`s than whenever an engineer might be trying to use `issue._related`.